### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: shared/components/ResourceNotFound/ResourceNotFound.js [hunk 1]
```diff
@@ -1,50 +1,22 @@
...
-          </div>
-        </section>
-      </header>
-      <LayoutPanel className="fd-margin--lg">
-        <LayoutPanel.Body className="fd-has-text-align-center">
-          <span className="fd-has-type-4">Such {resource} doesn't exists.</span>
-        </LayoutPanel.Body>
+      <PageHeader title="" breadcrumbItems={breadcrumbs} />
+      <LayoutPanel className="fd-has-padding-regular fd-margin--md">
+        {customMessage ? customMessage : `Such ${resource} doesn't exists.`}
```
Comment (dariadomagala) [inline shared/components/ResourceNotFound/ResourceNotFound.js:12]: I know it was like that before but.. :) 
```suggestion
        {customMessage ? customMessage : `Such ${resource} doesn't exist.`}
```

File: shared/utils/helpers.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { randomNamesGenerator } from './randomNamesGenerator/randomNamesGenerator';
+import { MESSAGES } from '../components/GenericList/constants';
```
Comment (dariadomagala) [inline shared/utils/helpers.js:2]: it's not used

File: shared/utils/helpers.js [hunk 3]
```diff
@@ -57,3 +58,14 @@ export const prettifyNameSingular = (resourceName, resourceType) => {
   const resources = prettifyNamePlural(resourceName, resourceType);
   return resources.slice(0, -1);
 };
+
+export const getErrorMessage = (error, message = null) => {
+  let errorNotification = message
+    ? message
+    : 'An error occured. The component cannot be rendered.';
+
+  if (error?.message && typeof error?.originalMessage !== 'object') {
```
Comment (dariadomagala) [inline shared/utils/helpers.js:67]: what's the 'originalMessage'?

Comment (akucharska) [reply shared/utils/helpers.js:67]: When we get a message it is stringified. So we need to check if it was an object or string before and that's what the originalMessage is for.

**Review by dariadomagala — DISMISSED** [review]

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**